tests/Testcontainers.Tests/Unit/Builders/Base64ProviderTest.cs

```diff
@@ -0,0 +1,65 @@
+using System.Text.Json;
+using DotNet.Testcontainers.Builders;
+using Microsoft.Extensions.Logging.Abstractions;
+using Xunit;
+
+namespace DotNet.Testcontainers.Tests.Unit
+{
+  public class Base64ProviderTest
```

Comment:
Can we  please move these tests to the already existing `Base64ProviderTest` test class? It is nested in `DockerRegistryAuthenticationProviderTest`. Probably we can reuse the existing test methods `ShouldGetAuthConfig` and `ShouldGetNull`.

Reply:
Absolutely, I got confused by the fact the existing `Base64ProviderTest` does not follow the same directory structure as in `src`. I have moved all the tests into the existing class in acd009ecea51ea89c9cbd232006a6bd426cd730c.